src/components/home/Testimonials.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Flex, Grid, Heading, Text } from "@radix-ui/themes";
+
+import { partnerTestimonials } from "@/data/home";
+
+const Testimonials = () => {
+  return (
+    <Box p="7">
+      <Box className="text-center" mb="8">
```

Comment:
I'd turn this box into a flex `<Flex direction="column" align="center" mb="8">` because there are more elements than just a bit of text.